src/edu/ucsb/projects/games/guess_the_capitals/Model/GameData.java

```diff
@@ -0,0 +1,153 @@
+
+package edu.ucsb.cs56.projects.games.guess_the_capitals;
+
+import java.util.Observable;
+import java.util.HashMap;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Holds all of the data for the game
+ *
+ * Also contains the question, and can determine whether or not an answer
+ * is right or wrong.
+ *
+ * @author Sean Shelton, Armin Mahini
+ * @version for UCSB CS56, F16  
+ */
+public class GameData extends Observable {
+	// TODO: Maybe consider making GameData a static singleton?
+
+	// Might want to consider making locationHashMap a JSON instead of a HashMap.
+	//private HashMap locationHashMap = new HashMap<Capital, Territory>();
+
+
+    //trying to implement HashMap refactoring here
+    private HashMap<String, ArrayList> games = new HashMap<String, ArrayList>();
+    private String names[] = {"US", "North America", "South America",
+			      "Europe", "Asia", "Africa", "Australia",
+			      "World", "All"};
+
+	/**
+	 *  File path for location of data
+	 */
+	private final static String filePath = "build/edu/ucsb/cs56/projects/games/guess_the_capitals/";
+
+	public static String getFilePath() {
+		return filePath;
+	}
+
+
+
+	/** Singleton instance to restrict instantiaion since we'll be using this GameData
+	 *  object across the whole game
+	 */
+	private static GameData instance = new GameData();
+
+	/** Constructs a newly allocated GameData object that knows all the territories and capitals for the
+	 *  US, world and both of those games.
+	 *  The controller can set the questionTerritory by putting in a capital, and then
+	 *  the questionTerritory will be the territory that the capital maps to in the HashMap.
+	 */
+	private GameData() {
+		// not sure what to have for default constructor yet,
+		// maybe default to the states HashMap?
+		File capitalsFile = new File(filePath + "capitals.txt");
```

Comment:
So instead of all of these different variables, I think it would be possible to just get all the files using [this method](https://docs.oracle.com/javase/7/docs/api/java/io/File.html#listFiles()), and just read each one.